src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -213,43 +227,6 @@ public inherited sharing class RD2_OpportunityEvaluationService {
         } set;
     }
 
-    /**
```

Comment:
Extracted the query builder from the `RD2_OpportunityEvaluationService` to untangle class dependency.